po/es.po

```diff
@@ -654,13 +654,13 @@ msgid ""
 "  -d, --lastday LAST_DAY        set date of last password change to "
 "LAST_DAY\n"
 msgstr ""
-"  -d, --lastday ÚLTIMO_DÍA      establece el día del último cambio de la\n"
-"                                contraseña a ÚLTIMO_DÍA\n"
+"  -d, --lastday ÚLTIMO_DÍA      establece a ÚLTIMO_DÍA el día del último\n"
+"                                cambio de la contraseña\n"
 
 msgid ""
 "  -E, --expiredate EXPIRE_DATE  set account expiration date to EXPIRE_DATE\n"
 msgstr ""
-"  -E, --expiredate FECHA_CAD    establece la fecha de caducidad a FECHA_CAD\n"
+"  -E, --expiredate FECHA_CAD    establece a FECHA_CAD la fecha de caducidad\n"
```

Comment:
I'd prefer to translate `expiration` as `expiración` instead of `caducidad`. In my opinion `caducidad` should be used for edibles and not for software related stuff.

Reply:
Seems fair